package.json

```diff
@@ -26,7 +26,7 @@
     "@angular/platform-browser": "~10.0.11",
     "@angular/platform-browser-dynamic": "~10.0.11",
     "@angular/router": "~10.0.11",
-    "@stoplight/elements-web-components": "beta",
+    "@stoplight/elements-web-components": "6.0.0-beta.163",
```

Comment:
This is pointing to a specific version only temporarly so we don't have `elements-stoplight-project` failing due to the changes from https://github.com/stoplightio/elements/pull/615. We can change it to `beta` later on